Formula/catch2.rb

```diff
@@ -17,11 +18,9 @@ class Catch2 < Formula
 
   depends_on "cmake" => :build
 
-  fails_with gcc: "5"
```

Comment:
This is needed for `c++17`

Reply:
That was put there in the first release of version 3, v3.0.1, because it failed to compile with GCC 5. Since then those bugs are fixed. The latest version v3.3.2 will compile with GCC 5 even if you request C++17.